### ISSUE TITLE
Panic when trying to generate

### ISSUE THREAD

**Issue by ryantimjohn** [dbt-labs/dbtcloud-terraforming#17]
```go
panic: interface conversion: interface {} is nil, not bool

goroutine 1 [running]:
github.com/dbt-labs/dbtcloud-terraforming/internal/app/dbtcloud-terraforming/cmd.init.generateResources.func2(0xc000142a00?, {0x162a918?, 0x4?, 0x162a91c?})
	/home/runner/work/dbtcloud-terraforming/dbtcloud-terraforming/internal/app/dbtcloud-terraforming/cmd/generate.go:173 +0x7ed9
github.com/spf13/cobra.(*Command).execute(0x1abfbe0, {0xc0000b9a40, 0x4, 0x4})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x863
github.com/spf13/cobra.(*Command).ExecuteC(0x1abf900)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/dbt-labs/dbtcloud-terraforming/internal/app/dbtcloud-terraforming/cmd.Execute()
	/home/runner/work/dbtcloud-terraforming/dbtcloud-terraforming/internal/app/dbtcloud-terraforming/cmd/root.go:30 +0x1a
main.main()
	/home/runner/work/dbtcloud-terraforming/dbtcloud-terraforming/cmd/dbtcloud-terraforming/main.go:8 +0xf
```
Hi all, got this while trying to generate a dbt cloud config.

**Comment by b-per** [comment]
HI! Thanks for reaching out.

- Could you share the command you ran?
- Also, did you specify a version for the provider in this config or left it as below?

```
terraform {
  required_providers {
    dbtcloud = {
      source = "dbt-labs/dbtcloud"
    }
  }
}
```

**Comment by b-per** [comment]
Looking at the dump I tried to guess where was the issue and did a fix in 0.7.1

Feel free to reopen if installing the latest version doesn't work.
